mlir/lib/Dialect/MIGraphX/MiGraphXTransform.cpp

```diff
@@ -63,7 +64,7 @@ struct MIGraphXTransforms
     RewritePatternSet patterns(&ctx);
     ConversionTarget target(ctx);
     target.addLegalDialect<migraphx::MIGraphXDialect, func::FuncDialect,
-                           tosa::TosaDialect>();
+                           tosa::TosaDialect, mhal::MHALDialect>();
```

Comment:
This does a full conversion as opposed to a partial conversion. 
https://mlir.llvm.org/docs/DialectConversion/

I think what you say is true only for a partial conversion.

Reply:
Ah, then yeah, that's everything.